src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/utilities/PointerLocation.java

```diff
@@ -272,18 +259,6 @@ public void printOn(PrintStream tty, boolean printAddress, boolean verbose) {
         getTLABThread().printThreadInfoOn(tty);
         tty.print(") ");
         getTLAB().printOn(tty); // includes "\n"
-      } else {
-        if (isInNewGen()) {
-          tty.print("In new generation ");
-        } else if (isInOldGen()) {
-          tty.print("In old generation ");
-        } else {
-          tty.print("In unknown section of Java heap");
-        }
-        if (getGeneration() != null) {
-          getGeneration().printOn(tty); // does not include "\n"
-        }
-        tty.println();
```

Comment:
If the pointer is in the heap, but not in the tlab, you don't print anything. But then due to the changes in PointerFinder, isInTLab() will always return false, so you won't print anything even if it is in a tlab.